spec/controllers/socure_webhook_controller_spec.rb

```diff
@@ -78,8 +80,26 @@
       expect(response).to have_http_status(:bad_request)
     end
 
+    context 'when DOCUMENTS_UPLOADED event received' do
+      let(:event_type) { 'DOCUMENTS_UPLOADED' }
+
+      context 'when document capture session does not exist' do
```

Comment:
can we add a test for when the document capture session does exist and verify that the job is queued (`SocureDocvResultsJob.perform_later` is called)?

Reply:
Done in b9a229bcc0